api/src/main/java/org/openmrs/module/locationbasedaccess/aop/interceptor/EncounterServiceInterceptorAdvice.java

```diff
@@ -37,7 +38,8 @@ public Object invoke(MethodInvocation invocation) throws Throwable {
         }
 
         Object object = invocation.proceed();
-        if (Daemon.isDaemonUser(authenticatedUser) || authenticatedUser.isSuperUser()) {
+        String lbacRestriction = Context.getAdministrationService().getGlobalProperty(LocationBasedAccessConstants.ENCOUNTER_RESTRICTION_GLOBAL_PROPERTY_NAME);
+        if (Daemon.isDaemonUser(authenticatedUser) || authenticatedUser.isSuperUser() || !("true".equals(lbacRestriction))) {
```

Comment:
the better option would be other way around right? 
`lbacRestriction.equals("true")`

Reply:
`"true".equals(lbacRestriction))` used it thinking misssing nullpointer exception by using `lbacRestriction.equals("true")` we can change it as lbacRestriction will not be NUll

Reply:
If you checking for null pointer expectation, then that needs to be diffrent. 

Here, it's means - you are checking where lbacRestriction is true or false. So the better option would be other way around. Is it?

Reply:
okay